### PR TITLE
Add support for /proc/net/softnet_stat

### PR DESCRIPTION
**Pull request by kormat**
softnet_stat contains important information for network performance
tuning. See these resources for details:
- https://blog.packagecloud.io/eng/2016/06/22/monitoring-tuning-linux-networking-stack-receiving-data/#monitoring-network-data-processing
- https://access.redhat.com/sites/default/files/attachments/20150325_network_performance_tuning.pdf
- https://github.com/leandromoreira/linux-network-performance-parameters#interrupt-coalescing-soft-irq-and-ingress-qdisc

### CODE REVIEW

File: net_softnet.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
+// Copyright 2019 Stephen Shirley <kormat@gmail.com>
```
Comment (pgier) [inline net_softnet.go]: Should be "The Prometheus Authors"

File: net_softnet_test.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Stephen Shirley <kormat@gmail.com>
```
Comment (pgier) [inline net_softnet_test.go]: Should be "The Prometheus Authors"

File: net_softnet.go (hunk 1 above)
Comment (kormat) [reply net_softnet.go]: Done.

File: net_softnet_test.go (hunk 2 above)
Comment (kormat) [reply net_softnet_test.go]: Done.

**Pull request merged**